pkg/controller/elasticsearch/nodespec/statefulset.go

```diff
@@ -116,21 +123,39 @@ func BuildStatefulSet(
 
 func setVolumeClaimsControllerReference(
 	persistentVolumeClaims []corev1.PersistentVolumeClaim,
+	existingClaims []corev1.PersistentVolumeClaim,
```

Comment:
nit: `inheritedClaims` vs `existingClaims`, should we use only one term? I like `existingClaims`, it's explicit.